pax-chassis-web/interface/src/classes/native-element-pool.ts

```diff
@@ -276,6 +276,10 @@ export class NativeElementPool {
         if (patch.transform != null) {
             leaf.style.transform = packAffineCoeffsIntoMatrix3DString(patch.transform);
         }
+
+        if (patch.focus_on_mount) {
+            setTimeout(() => { textbox.focus(); console.log("focused!"); }, 10);
```

Comment:
any reason we set a timeout here rather than just call textbox.focus()?
also we can remove the console.log

Reply:
Oops, Removed log!

Didn't focus correctly for some reason without it, not exactly sure why. It should have been added to the dom already? (requirement for focus to work). I just took the fast path :)

Reply:
cool that sounds good. We can figure out later.